common/pom.xml

```diff
@@ -18,6 +18,18 @@
         <jackson.modules.version>2.8.5</jackson.modules.version>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
```

Comment:
Can we add a comment in here that briefly says this since it deviates from our other dependencies?